BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -210,35 +210,11 @@ extension AttributedLabel {
             }
         }
 
-        /// The view needs to keep track of the current index for the accessibility rotor.
-        private var accessibilityLinkIndex = -1
-
-        /// These elements need to be retained by the view, and cannot be created inside the
-        /// `accessibilityCustomRotors` getter.
-        private var accessibilityLinks: [LinkAccessibilityElement] = []
-
         override var accessibilityCustomRotors: [UIAccessibilityCustomRotor]? {
             set { fatalError() }
```

Comment:
Nit (and outside the scope of the PR): would it be better have this be a no-op instead of a crasher?
```suggestion
            set { _ = newValue }
```

Reply:
The pattern in the rest of Blueprint is to `fatalError` for "read only" property overrides, so I'd like to match that. But I should add a more descriptive message.

Reply:
That makes sense. I generally try to avoid fatalErrors at all cost, but if this is the pattern for this repo, it makes sense to stick to it. At some point it might be worth discussing making a systemic change - but certainly not now!